src/main/java/fr/inria/coming/codefeatures/FeatureAnalyzer.java

```diff
@@ -126,10 +126,17 @@ public AnalysisResult analyze(IRevision revision, RevisionResult previousResults
 
 				changesArray.add(new Gson().fromJson(analyze.toJson().toString(), JsonObject.class));
 				tempFile.delete();
+				
+				//add more features
+				JsonObject patternJson = RepairPatternFeatureAnalyzer.analyze(revision.getFolder(), diff, nameFile.toString());
+				changesArray.add(patternJson);
+				
 			} catch (Exception e) {
 				new RuntimeException("Unable to compute ADD analysis", e);
```

Comment:
I detect that this code is problematic. According to the [Correctness (CORRECTNESS)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#correctness-correctness), [RV: Exception created and dropped rather than thrown (RV_EXCEPTION_NOT_THROWN)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#rv-exception-created-and-dropped-rather-than-thrown-rv-exception-not-thrown).
This code creates an exception (or error) object, but doesn't do anything with it. For example, something like
It was probably the intent of the programmer to throw the created exception: